partcad/src/partcad/user_config.py

```
@@ -10,143 +10,110 @@
+                with open(config_path, "r") as f:
+                    self.read_config(f)
             except Exception as e:
                 pc_logging.error("ERROR: Failed to parse %s" % config_path)
 
+        if shutil.which("conda") is not None or importlib.util.find_spec("conda") is not None:
+            self.set_default("pythonSandbox", "conda")
+        else:
+            self.set_default("pythonSandbox", "none")
+
+        self.set_default("internalStateDir", UserConfig.get_config_dir())
+        self.set_default("forceUpdate", False)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)